plantcv/plantcv/pseudocolor.py

```diff
@@ -108,38 +115,52 @@ def pseudocolor(gray_img, mask=None, cmap=None, background="image", min_value=0,
         # Plot the background first
         pseudo_img1 = plt.imshow(bkg_img, cmap=bkg_cmap)
```

Comment:
```suggestion
        pseudo_img1 = plt.imshow(bkg_img, cmap=bkg_cmap, vmin=min_value, vmax=max_value)
```

Reply:
I think we need to apply the same vmin and vmax to the background...but not 100% sure.